### PR TITLE
Add homebrew-installed completions to fpath

### PR DESCRIPTION
**Pull request by jedahan**
Used the node submodule as inspiration.

### CODE REVIEW

File: modules/homebrew/init.zsh [hunk 1]
```diff
@@ -31,3 +31,7 @@ alias caski='brew cask install'
 alias caskl='brew cask list'
 alias casks='brew cask search'
 alias caskx='brew cask uninstall'
+
+if (( $+commands[brew] )) && [[ -d "$(brew --prefix 2>/dev/null)" ]]; then
```
Comment (sorin-ionescu) [inline modules/homebrew/init.zsh]: Cache the result of `brew --prefix`; don't call it twice. Can you have `brew` without being installed?

**Comment by jedahan** [comment]
Is this a bit better?

Would we want to change the node module to work the same? https://github.com/sorin-ionescu/prezto/blob/master/modules/node/init.zsh#L14-L15

**Comment by sorin-ionescu** [comment]
Yes, `brew` is written for Ruby 1.8, which is slow. Calling it twice is not good. A separate pull request should be opened for that.

**Comment by jedahan** [comment]
No longer using prezto, so can't test this anymore

**Pull request closed without merging**